Source/gui/SNTNotificationManager.m

```diff
@@ -100,7 +100,20 @@ - (void)queueMessage:(SNTMessageWindowController *)pendingMsg {
 
   // If GUI is in silent mode or if there's already a notification queued for
   // this message, don't do anything else.
-  if ([SNTConfigurator configurator].enableSilentMode) return;
+  if ([SNTConfigurator configurator].enableSilentMode) {
+    // hash bundle with path
+    dispatch_async(self.hashBundleBinariesQueue, ^{
+          SNTBinaryMessageWindowController *wc = (SNTBinaryMessageWindowController *)pendingMsg;
```

Comment:
The indentation inside this block should be +2 spaces from the previous line (i.e. -4 spaces from where it is now).

Running `./Testing/fix.sh` from your checked out repo should fix this and other style issues automatically.

Reply:
fixed with `./Testing/fix.sh`